### DISCUSSION TITLE
초기 플레이어(캐릭터) 위치 수정

### DISCUSSION THREAD

**Discussion by elimieum** [naverz/zepeto-studio-kor#246] [5. Q&A]
안녕하세요
지금 유니티에서 월드를 만드는 중입니다.
상단의 플레이 버튼을 누르면 캐릭터가 땅속에 묻혀 나타납니다.
그렇다고 맵제작 특징상 맵위치를 조정할수는 없고
JepeotoPlayer 위치를 이동해도 플레이를 하면 항상 같은 위치에서 로딩이 됩니다.

유니티에서 플레이버튼을 클릭하면 초기에 불러오게되는 캐릭터의 위치를 수정하려면 어떻게 하나요?

**Comment by x4cf** [comment]
도움이 될 수 있을까요?
https://github.com/naverz/zepeto-studio-kor/discussions/196

**Reply by elimieum** [reply]
이렇게 도움들 주시어 감사합니다.^^

**Comment by elimieum** [comment]
아래 첨부이미지 스폰인포 스크립트를 적용하라는 의미가 맞는지요?

<img width="1026" alt="spwan" src="https://user-images.githubusercontent.com/97677372/150895262-b00382f7-4101-47e8-8e2b-3db68616a36d.png">

맞다면 첨부된 스크립트를
ZepetoPlayers 또는 GameObject중 어디에다 연결하면 되나요?

위내용이 틀리다면
방법을 알려주시면 고맙겠습니다.방법을 알려주시면 고맙겠습니다.^^

**Reply by x4cf** [reply]
깃 G메일 알람이 겹치면 잘 안보이네요, 지금 확인했습니다.
아래에 좋은 답변대로 GameObject 에 넣는것이 좋으며, 저는 제페토 멀티플레이 예시프로젝트의 ClientStarter 스크립트 안에 적재합니다.

**Comment by hyeonjin-jo** [comment]
안녕하세요.

초기 플레이어 위치를 결정해서 생성하는 코드는 elimieum님이 위에 스크린샷 올려주신 부분 처럼  스폰 인포를 변경해주시면 됩니다.

스크립트 연결은 아무곳이나 가능하나 가독성을 위해서 ZepetoPlayers오브젝트와는 별도의 GameObject를 만드는 것을 추천드립니다.

해당함수를 호출하는 자세한 방법은 아래의 링크의 스크립트를 참고해 주세요.

https://github.com/naverz/zepeto-multiplay-example/blob/main/Assets/ZepetoScripts/ClientStarter.ts

**Reply by elimieum** [reply]
이렇게 도움들 주시어 감사합니다.^^
